code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -100,23 +100,24 @@
 
 	inherent_traits = list(
 		// SHARED WITH ALL ZOMBIES
+		TRAIT_AGENDER,
 		TRAIT_EASILY_WOUNDED,
 		TRAIT_EASYDISMEMBER,
 		TRAIT_FAKEDEATH,
 		TRAIT_LIMBATTACHMENT,
+		TRAIT_LIVERLESS_METABOLISM,
 		TRAIT_NOBREATH,
 		TRAIT_NOCLONELOSS,
 		TRAIT_NODEATH,
 		TRAIT_NOHUNGER,
-		TRAIT_LIVERLESS_METABOLISM,
+		TRAIT_NO_TRANSFORMATION_STING,
 		TRAIT_RADIMMUNE,
 		TRAIT_RESISTCOLD,
 		TRAIT_RESISTHIGHPRESSURE,
 		TRAIT_RESISTLOWPRESSURE,
 		TRAIT_TOXIMMUNE,
 		// INFECTIOUS UNIQUE
 		TRAIT_STABLEHEART, // Replacement for noblood. Infectious zombies can bleed but don't need their heart.
-		TRAIT_STABLELIVER, // Not necessary but for consistency with above
```

Comment:
zombies have TRAIT_LIVERLESS_METABOLISM, this subtype doesn't have a mutantliver, this makes zero sense and i don't like it